content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -22,11 +22,82 @@ You must use a kubectl version that is within one minor version difference of yo
 
```

Comment:
Overview section: Suggest revising the first sentence to: The Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/), allows you to deploy and manage applications on Kubernetes.